### DISCUSSION TITLE
Compilation issues of nrsconfig

### DISCUSSION THREAD

**Discussion by MingTengCA** [Nek5000/nekRS#517] [Q&A]
Hey guys,

I ran into an compiling issue when installing the nekRS on the HPC. here it is:

![image](https://github.com/Nek5000/nekRS/assets/142837254/c201ebd1-58fc-43cb-b8e0-1b7caf33e0b3)

One error shown here (at config/nek5000.cmake:9) is: no supported Fortran compiles found to build Nek5000 interface. However, I do have LLVM loaded.

thanks for you help,
Ming

**Comment by yslan** [comment]
Hi Ming,

Is your mpif77 the right mpi fortran wrapper and link to the correct fortran compiler?

**Reply by MingTengCA** [reply]
Thank you Yu-Hsiang,

I think they are all good.
So far, a few errors have been sorted out. The remaining one seems to be from HIP:

-- Could NOT find HIP (missing: HIP_ROOT_DIR HIP_INCLUDE_DIRS HIP_LIBRARIES HIP_RUNTIME_DEFINE HIP_HIPCONFIG_EXECUTABLE HIP_PLATFORM HIP_COMPILER)

HPC people is currently looking into this issue.

thanks again,
Ming

**Comment by stgeke** [comment]
LLVM‘s Fortran compiler is currently not supported.

**Reply by MingTengCA** [reply]
I see, good to know.
Thank you!
